src/integrations/GA4/browser.js

```diff
@@ -24,10 +24,11 @@ export default class GA4 {
     this.measurementId = config.measurementId;
     this.capturePageView = config.capturePageView || 'rs';
     this.overrideSessionId = config.overrideSessionId || false;
-    this.areTransformationsConnected = destinationInfo && destinationInfo.areTransformationsConnected;
+    this.areTransformationsConnected =
+      destinationInfo && destinationInfo.areTransformationsConnected;
     this.destinationId = destinationInfo && destinationInfo.destinationId;
     this.extendPageViewParams = config.extendPageViewParams || false;
-    this.isHybridModeEnabled = config.useNativeSDKToSend === false || false;
+    this.isHybridModeEnabled = config.connectionMode === 'hybrid';
```

Comment:
all destination configs are migrated to `connectionMode` instead of `useNativeSDKToSend`

Reply:
all the new form builder destinations will have `connectionMode` key support, in old ui it's not supported yet

Reply:
we should handle both in the condition

Reply:
we don't need that as we already migrated all existing users to new form builder so `connectionMode` key will always be present

ref [this](https://github.com/rudderlabs/rudder-config-backend/blob/267049e6be6b9a7ada3204e1ef62635f353ce92f/src/services/scripts.service.ts)